crates/common/src/utils.rs

```diff
@@ -273,6 +281,158 @@ pub fn get_user_agent_with_version(req_headers: &HeaderMap) -> eyre::Result<Head
     Ok(HeaderValue::from_str(&format!("commit-boost/{HEADER_VERSION_VALUE} {}", ua))?)
 }
 
+/// Parse ACCEPT header, default to JSON if missing or mal-formatted
+pub fn get_accept_header(req_headers: &HeaderMap) -> Accept {
+    Accept::from_str(
+        req_headers.get(ACCEPT).and_then(|value| value.to_str().ok()).unwrap_or("application/json"),
+    )
+    .unwrap_or(Accept::Json)
+}
+
+/// Parse CONTENT TYPE header, default to JSON if missing or mal-formatted
+pub fn get_content_type_header(req_headers: &HeaderMap) -> ContentType {
+    ContentType::from_str(
+        req_headers.get(CONTENT_TYPE).and_then(|value| value.to_str().ok()).unwrap_or("application/json"),
+    )
+    .unwrap_or(ContentType::Json)
+}
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum ContentType {
+    Json,
+    Ssz
+}
```

Comment:
Just a nit, but I would rename this enum to `MediaType` which is most accurate term.

Reply:
Well actually, I understand what you're doing now. You're splitting this into `ContentType` and `Accept`. I would probably combine these into a `MediaType` enum and try to not have an `Any` value.